### DISCUSSION TITLE
Can you change the version of every mod at once?

### DISCUSSION THREAD

**Discussion by GamingIQ** [PrismLauncher/PrismLauncher#3243] [Q&A] [answered]
I downgraded my instance to 1.20.1 and wanted to select and downgrade every mod, but the change version button didn't work.

**Comment by om3ga6400** [comment]
dose the update mods button change them to the other mc ver?

**Reply by GamingIQ** [reply, marked as answer]
bro its been a month why are you commenting

but ye it does i think

**Reply by om3ga6400** [reply]
> bro its been a month why are you commenting
> 
> but ye it does i think

because I was looking at the discussions and saw that this was unanswered

**Reply by om3ga6400** [reply]
also @GamingIQ make sure to close this as answered
